### DISCUSSION TITLE
Sync Imported playlists

### DISCUSSION THREAD

**Discussion by aneesh1122** [HemantKArya/BloomeeTunes#32] [Ideas]
It would be great if there was an option to sync playlists imported from YouTube or Spotify.

This way we can easily update our imported playlists if there are any changes in the original playlists

**Comment by HemantKArya** [comment]
Yeah, sure thanks for noticing that @aneesh1122 . I marked this as feature request so we can track this thread for implementation.

**Comment by aneesh1122** [comment]
Converted this to a discussion?

![magar-kyu-krk.gif](https://github.com/user-attachments/assets/4fec82b0-a58d-40bd-99cb-0cc88e4f4fb5)

**Reply by HemantKArya** [reply]
yup 😂 because right now I think it need more features so I just converted it to discussion and Ideas categories, don't worry I am still working on it, and you can watch some commits related to this issue are already in stable version like persist the link as metadata for imported playlist and albums so that we can update it later. But right now, I think we need more discussion (if anyone wants.) to make this feature work more efficiently.

**Comment by BossDaily** [comment]
Any progress on this?

**Comment by Korgman19** [comment]
Is there a possibility to stream it to an Alexa or Google Assistant device without using Bluetooth? That would be a game-changer, also using a web-based player
